src/jobTree/test/jobStores/jobStoreTest.py

```diff
@@ -225,6 +225,22 @@ def test( self ):
             # Delete a file explicitly but leave files for the implicit deletion through the parent
             worker.deleteFile( fileOne )
 
+            # Test stats and logging
+            testRead = []
+            files=master.readStatsAndLogging(testRead.append)
+            assert files==0
```

Comment:
We should either use the `assert` statement or the `self.assertXyz` methods but not both. I prefer the latter since they give better diffs in my IDE.